var/spack/repos/builtin/packages/genometools/package.py

```diff
@@ -13,14 +13,16 @@ class Genometools(MakefilePackage):
     homepage = "http://genometools.org/"
     url      = "http://genometools.org/pub/genometools-1.5.9.tar.gz"
 
+    version('1.6.1', sha256='528ca143a7f1d42af8614d60ea1e5518012913a23526d82e434f0dad2e2d863f',
+            url="https://github.com/genometools/genometools/archive/v1.6.1.tar.gz")
```

Comment:
Can we move the package URL to Github, so that we don't need to specify it version by version?

Reply:
fixed. Hash of older version changed as a result.